README.md

```diff
@@ -0,0 +1,17 @@
+# Contacts Manager web app
+
+## Requirements:
+- python flask
+- pytest 
+- selenium 
+
+## Run the app
+- clone the repo 
+- run `flask run` at the root level
+- open `http://127.0.0.1:5000/` in a browser
+
+## Run the tests
+- go to `/tests`
+- run `pytest`
+
+![app screenshot of app](https://image.ibb.co/iTDrv9/Zrzut_ekranu_2018_08_30_o_14_55_24.png
```

Comment:
nitpick: I think you need to close the round parens

Reply:
yep, fixed it already on master :)